include/sys/poll.h

```diff
@@ -78,11 +78,6 @@
 #define POLLRDHUP    (0x10)  /* NuttX does not support shutdown(fd, SHUT_RD) */
```

Comment:
Seeing this previous not supported comment, I have a question: how to implement this missing shutdown at libs/libc/net/lib_shutdown.c ?

Reply:
it's impossible, I think. The real implementation need move to net/